ethcore/blockchain/src/blockchain.rs

```diff
@@ -963,6 +963,7 @@ impl BlockChain {
 	/// Iterate over all epoch transitions.
 	/// This will only return transitions within the canonical chain.
 	pub fn epoch_transitions(&self) -> EpochTransitionIter {
+		trace!(target: "blockchain", "Iterating over all epoch transitions");
```

Comment:
```suggestion
		debug!(target: "blockchain", "Iterating over all epoch transitions");
```

Might be helpful if we collect only `debug` logs initially.